api/end_device.proto

```diff
@@ -610,6 +610,13 @@ message SetEndDeviceRequest {
   google.protobuf.FieldMask field_mask = 2 [(gogoproto.nullable) = false];
 }
 
+message ResetEndDeviceRequest {
+  EndDeviceIdentifiers end_device_ids = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false, (validate.rules).message.required = true];
+  // The names of the end device fields that should be returned.
+  // See the API reference for which fields can be returned by the different services.
+  google.protobuf.FieldMask field_mask = 2 [(gogoproto.nullable) = false];
+}
```

Comment:
Even though it's documented that `field_mask` is for the "get" part, it's not clear otherwise that there's something to get. API clients may assume that the field mask indicates what is being reset.

Suggesting `RestAndGetEndDeviceRequest` or something